netkan/Dockerfile

```diff
@@ -5,6 +5,7 @@ WORKDIR /netkan
 RUN chown -R netkan:netkan /netkan
 USER netkan
 RUN pip install --user . --no-warn-script-location
+RUN /home/netkan/.local/bin/mypy .
 RUN python -m unittest -v
```

Comment:
I'm assuming mypy is only used in tests? We could add a test section in our extra requires then pip install them globally in this stage. 

Like this
```
RUN pip install /netkan/.[test]
RUN /home/netkan/.local/bin/mypy .
￼RUN python -m unittest -v
```
Then these deps won't end up in our production container. The Dev stage will need them installed as well.

Reply:
That sounds cool! I don't think I have the requisite Docker expertise, though. Do you want to take this on?

Reply:
Also would that give us the option to leave the `tests` folder out of the production image?

Reply:
That already happens as part of the setup.py build. It's why the production stage uses a --copy-from, to grab only the installation.

I think we should add a testing stage, but that would've taken longer than a coffee break to sort out. But I've done what will work for now and keeps mypy out of our prod image.

I didn't use the method I suggested, as that installs all the deps + the test one to global, which isn't ideal.

Reply:
Nice. I missed that the `COPY --from` allows us to install stuff that won't carry forward as long as it's not in `/home/netkan`. That solves this nicely.